Classes/Service/ConfigurationService.php

```diff
@@ -81,6 +81,11 @@ class ConfigurationService extends FluxService implements SingletonInterface {
 	protected $pageUidBackup;
 
 	/**
+	 * @var array
+	 */
+	protected $wizardtabIcons;
```

Comment:
We should avoid this storage - it's not directly required by the method below (the array can be cached and retrieved without being a class property) and the container class is a Singleton which means we should as far as possible avoid mutating its state.

If icons then need to be cached individually that's okay too ;)